modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -133,6 +133,16 @@
 		"newReference": "HtmlParser _htmlParser",
 		"to": "_htmlParser.extractText"
 	},
+	{
+		"classNames": [
+			"CommerceAddressService",
+			"CommerceAddressLocalService"
```

Comment:
@MicaelleSilvaa, can you please sort alphabetically?

In addition, can you please double check if the method `updateCommerceAddress` is also in the classes `CommerceAddressServiceUtil` and `CommerceAddressLocalServiceUtil`?

If so, can you please include them here and propagate it to the tests?

You can check similar checks using the `*Util` classes in this [file](https://github.com/natocesarrego/liferay-portal/blob/3521e8997a75d7793e3448d0c7438c79d1b486e9/modules/util/source-formatter/src/main/resources/dependencies/replacements.json#L156-L161). If you need some assistance, please let me know.

Reply:
Sorry, I hadn't checked before. I checked the updateCommerceAddress method and it is only found in the CommerceAddressLocalServiceUtil and CommerceAddressLocalService classes.